macros/CSSRef.ejs

```diff
@@ -81,8 +81,11 @@ if (slug) {
             group = group.concat(data.atRules[name].groups);
         } else if (data.properties[name] && data.properties[name].groups) {
             group = group.concat(data.properties[name].groups);
-        } else if (isType(title) && data.types[title] && data.types[title].groups) {
-            group = group.concat(data.types[title].groups);
+        } else if (isType(title)) {
+          var cleanType = title.replace('<', '').replace('>', '');
+          if (data.types[cleanType] && data.types[cleanType].groups) {
+            group = group.concat(data.types[cleanType].groups);
+          }
```

Comment:
This will require synchronizing the merging and deployment of this and https://github.com/mdn/data/pull/81, which seems doomed to failure.  What if we make this code work with both versions?

```
        } else if (isType(title)) {
            var cleanType = title;
            if (!(cleanType in data.types)) {
                // https://github.com/mdn/data/pull/81 was merged
                cleanType = title.replace('<', '').replace('>', '');
            }
            if (data.types[cleanType] && data.types[cleanType].groups) {
                group = group.concat(data.types[cleanType].groups);
            }
        }
```

I was going to complain about indenting, but it appears there's no standard.